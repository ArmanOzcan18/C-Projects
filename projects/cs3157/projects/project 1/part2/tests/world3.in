invalid
test
entries
are
put
in
and
they
do
not
cost
guesses
