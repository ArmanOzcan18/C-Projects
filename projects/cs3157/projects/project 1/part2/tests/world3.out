Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Your guess must be exactly 5 characters. Please make another.
Guess:Game aborted. The answer was: world
